### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by AustinSeto**

### CODE REVIEW

**Comment by priyyanshi** [comment]
Tested locally and passed.

File: README.adoc [hunk 1]
```diff
@@ -33,14 +33,18 @@ Learn how to use a reactive JAX-RS client to asynchronously invoke RESTful micro
 
 == What you'll learn
 
-You'll first learn how to create a reactive JAX-RS client application using the default JAX-RS reactive provider APIs. You will then learn how to improve the application to take advantage of the RxJava reactive extensions with a pluggable reactive provider that's published by https://eclipse-ee4j.github.io/jersey[Eclipse Jersey^].
+You'll first learn how to create a reactive JAX-RS client application using the default JAX-RS reactive provider APIs.
```
Comment (dmuelle) [inline README.adoc]: You'll first learn how to create a reactive JAX-RS client application using the default JAX-RS reactive provider APIs.
--->
First, you'll learn how to create a reactive JAX-RS client application by using the default JAX-RS reactive provider APIs.

Comment (dmuelle) [inline README.adoc]: What are " the default JAX-RS reactive provider APIs."? When I search that term, I get results for  "JAX-RS client  APIs." If they are thee same thing, recommend you replace provider with client in above sentence. If not, disregard this comment

File: README.adoc [hunk 2]
```diff
@@ -33,14 +33,18 @@ Learn how to use a reactive JAX-RS client to asynchronously invoke RESTful micro
...
+You will then learn how to improve the application to take advantage of the RxJava reactive extensions with a
```
Comment (dmuelle) [inline README.adoc]: You will then learn how to improve the application to take advantage of the RxJava reactive extensions with a
pluggable reactive provider that's published by https://eclipse-ee4j.github.io/jersey[Eclipse Jersey^].

--->

You will then learn how to take advantage of the RxJava reactive extensions with a
pluggable reactive provider that's published by https://eclipse-ee4j.github.io/jersey[Eclipse Jersey^].

see previous comment re provider vs client

File: README.adoc [hunk 3]
```diff
@@ -33,14 +33,18 @@ Learn how to use a reactive JAX-RS client to asynchronously invoke RESTful micro
...
+pluggable reactive provider that's published by https://eclipse-ee4j.github.io/jersey[Eclipse Jersey^].
 The JAX-RS client is an API used to communicate with RESTful web services. 
-The API makes it easy to consume a web service that is exposed by using the HTTP protocol, which means that you can efficiently implement client-side applications. 
+The API makes it easy to consume a web service that is exposed by using the HTTP protocol,
```
Comment (dmuelle) [inline README.adoc]: "by using the HTTP protocol,"

does this refer to "consume a web service by using the HTTP protocol" or "service is exposed by using the HTTP protocol" ? Could be read either way

File: README.adoc [hunk 4]
```diff
@@ -75,29 +82,31 @@ include::{common-includes}/gitclone.adoc[]
...
-----
-include::defaultrx/query/src/main/java/io/openliberty/guides/query/client/InventoryClient.java[]
-----
-
 Navigate to the `start` directory to begin.
 
 JAX-RS provides a default reactive provider that you can use to create a reactive REST client using the `CompletionStage` interface.
 
-Create an [hotspot file=0]`InventoryClient` class, which is used to retrieve inventory data, and a [hotspot file=1]`QueryResource` class, which queries data from the `inventory` service.
+Create an [hotspot file=0]`InventoryClient` class, which is used to retrieve inventory data,
```
Comment (dmuelle) [inline README.adoc]: which is used to retrieve inventory data,
--->
which retrieves inventory data,

File: README.adoc [hunk 5]
```diff
@@ -168,8 +183,36 @@ include::{common-includes}/os-tabs.adoc[]
...
+curl -s http://localhost:9085/health | jq
+```
+{: codeblock}
+
+```
+curl -s http://localhost:9080/health | jq
+```
+{: codeblock}
+
+Once the microservices are up and running, you can access the application by making requests to the **query/systemLoad** endpoint
```
Comment (dmuelle) [inline README.adoc:209]: ...using the following **curl** command:
--->
...by using the following **curl** command:

**Review by dmuelle — CHANGES_REQUESTED** [review]
A few suggestions-  want to confirm that "JAX RS reactive provider API" is the correct term and not  "JAX RS reactive client API", which appears to be in more common usage.

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed

Comment (gkwan-ibm) [reply README.adoc]: addressed.
Is the change better?

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed.
Is the change better?

File: README.adoc (hunk 3 above)
Comment (gkwan-ibm) [reply README.adoc]: In my understanding, the client invokes HTTP calls to the services. 
Can I answer your question?

File: README.adoc (hunk 4 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed

File: README.adoc (hunk 5 above)
Comment (gkwan-ibm) [reply README.adoc:209]: addressed

File: README.adoc (hunk 3 above)
Comment (dmuelle) [reply README.adoc]: would the meaning be the same if it was:

The API makes it easy to consume a web service by using the HTTP protocol, which means that you can efficiently implement client-side applications.

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**